homeassistant/components/roku/binary_sensor.py

```diff
@@ -0,0 +1,93 @@
+"""Support for Roku binary sensors."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from rokuecp.models import Device as RokuDevice
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .entity import RokuEntity
+
+
+@dataclass
+class RokuBinarySensorEntityDescriptionMixin:
+    """Mixin for required keys."""
+
+    is_on_fn: Callable[[RokuDevice], bool | None]
```

Comment:
For consistency, I believe these are usually named `value_fn`

https://github.com/home-assistant/core/blob/dev/homeassistant/components/august/binary_sensor.py#L105

Reply:
thanks. Updated